esp/src/eclwatch/FilterDropDownWidget.js

```diff
@@ -105,13 +107,13 @@ define([
             return false;
         },
 
-        toObject: function () {
+        toObject: function (filter) {
             var context = this;
             if (this.filterDropDown.get("disabled")) {
                 return {};
             }
             var retVal = {};
-            arrayUtil.forEach(this.filterForm.getDescendants(), function (item, idx) {
+            arrayUtil.forEach(this.filterForm.getDescendants() || filter, function (item, idx) {
```

Comment:
When does `filter` actually come into play?  Doesn't `this.filterForm.getDescendants()` always return something?

Reply:
When a user physically click apply. Rather than different publishers, queries that call toObject()